hardware/raspberrypi/bcm2711_bootloader_config.md

```diff
@@ -301,75 +335,15 @@ N.B `flashrom` does not support clearing of the write-protect regions and will f
-
-### Using Raspberry Pi Imager to update the bootloader (recommended)
-Raspberry Pi Imager provides a GUI for updating the bootloader
-
-1. Download [Raspberry Pi Imager](https://www.raspberrypi.org/downloads/)
-2. Select a spare SD card. The contents will get overwritten!
-3. Launch `Raspberry Pi Imager`
-4. Select `Misc utility images` under `Operating System`
-5. Select `Bootloader`
-6. Select a boot-mode 
-7. Select `SD card` and then `Write`
-
-### Using raspi-config to update the bootloader from within Raspberry Pi OS
-To change the boot-mode or bootloader version from within Raspberry Pi OS run [raspi-config](../../configuration/raspi-config.md)
-
-1. Run `sudo raspi-config`
-2. Select `Advanced Options`
-3. Select `Boot Order`
-4. Select SD, USB or Network boot
-5. Reboot
-
-### Custom boot-order
-The `rpi-eeprom-config` utility has an editor mode which can be used to make quick changes to the EEPROM config file. Use this to define the `BOOT_ORDER` setting directly
-
-```
-sudo apt update
-sudo apt full-upgrade
-sudo -E rpi-eeprom-config --edit
-```
-Make changes, then save and quit. `rpi-eeprom-config` will print details of the EEPROM release and how to revert changes. Reboot to apply the bootloader update.
-
-The [Boot EEPROM](booteeprom.md) page provides more information about the EEPROM flashing process.
-
-
-#### Hardware compatibility for USB mass storage devices
-Before attempting to boot from a USB mass storage device it is advisible to verify that the device works correctly under Linux. Boot using an SD card and plug in the USB mass storage device. This should appears as a removable drive.
-
-*Spinning hard-disk drives nearly always require a powered USB hub. Even if it appears to work you are likely to encounter intermittent failures without a powered USB HUB*
-
-This is especially important with USB SATA adapters which may be supported by the bootloader in mass storage mode but fail if Linux selects [USB Attached SCSI - UAS](https://en.wikipedia.org/wiki/USB_Attached_SCSI) mode.
+<a name="config_txt"></a>
+### config.txt section
+From Raspberry Pi OS 2021-01-11, the firmware now reads the bootloader configuration from the EEPROM. If a section called `[config.txt]` is found then the contents from the start of this section to the end of the file is appended in memory, to the contents of the `config.txt` file read from the boot partition.  This can be used to automatically apply settings to every operating system, for example, dtoverlays.
 
-See this [forum thread](https://www.raspberrypi.org/forums/viewtopic.php?t=245931) about UAS and how to add [usb-storage.quirks](https://www.kernel.org/doc/html/v5.0/admin-guide/kernel-parameters.html) to workaround this issue.
+If an invalid configuration which causes boot to fail is specified then an EEPROM recovery image or usbboot will be required to restore the EEPROM to factory defaults.
```

Comment:
Perhaps the "usbboot" here should be changed to "RPIBOOT" in order to avoid any confusion?

Reply:
Yeah, probably. usbboot is used quite a lot throughout the docs but I think we should use RPIBOOT everywhere to mean the ROM USB device boot.

Reply:
Yeah, IIRC usbboot was originally created for flashing the eMMC on a CM1, and at that time booting a Pi from a USB MSD wasn't even a thing, so the 'usbboot' name was much less confusing then, than it could be now :wink:

Reply:
done